packages/yew/src/html/component/lifecycle.rs

```diff
@@ -236,6 +233,18 @@ impl<COMP: BaseComponent> Runnable for RenderRunner<COMP> {
 
                         let node = new_root.apply(&scope, m, next_sibling, ancestor);
                         state.node_ref.link(node);
+
+                        let first_render = !state.has_rendered;
+                        state.has_rendered = true;
+
+                        scheduler::push_component_rendered(
+                            self.state.as_ptr() as usize,
+                            RenderedRunner {
+                                state: self.state.clone(),
+                                first_render,
+                            },
+                            first_render,
```

Comment:
I don't like how the same thing is passed twice but we don't have a choice, do we? Can't extract `first_render` from `impl Runnable`

Reply:
It's not impossible as long as we make `push_component_rendered` to accept `RenderedRunner` only so it can access struct fields.

But I don't think that change is needed / has any actual benefits (and for the sake of consistency, we need to change all `push_*` methods).

Reply:
I agree. It's maybe just a couple of bytes more which doesn't really matter